docs/setup.md

```diff
@@ -2,7 +2,146 @@
 title: 'Setup'
 ---
 
-# GitHub Actions [linux/mac/windows] {: #github-actions}
+# Build your wheel locally
```

Comment:
This should be a new page. This page talks about running cibuildwheel on CI (and locally), not about setuptools details.

Reply:
Yeah coming back to this, without a large-ish change to docs structure, I think you're right.

Reply:
I've reworded it and moved the section to Tips and tricks, as 'Alternatives to cibuildwheel options'. Suggestions welcome on that title, though!